src/Instrumentation/Yii/composer.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "open-telemetry/opentelemetry-auto-yii",
+    "description": "OpenTelemetry auto-instrumentation for Yii",
+    "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "yii", "instrumentation"],
+    "type": "library",
+    "homepage": "https://opentelemetry.io/docs/php",
+    "readme": "./README.md",
+    "license": "Apache-2.0",
+    "minimum-stability": "dev",
+    "require": {
+      "php": "^8.0",
+      "ext-opentelemetry": "*",
+      "yiisoft/yii2": "^2.0.13",
+      "open-telemetry/api": "^1.0.0beta10",
+      "open-telemetry/sem-conv": "^1.22"
+    },
+    "require-dev": {
+      "friendsofphp/php-cs-fixer": "^3",
+      "open-telemetry/sdk": "^1.0",
+      "phan/phan": "^5.0",
+      "php-http/mock-client": "*",
+      "phpstan/phpstan": "^1.1",
+      "phpstan/phpstan-phpunit": "^1.0",
+      "psalm/plugin-phpunit": "^0.16",
+      "phpunit/phpunit": "^9.5",
+      "vimeo/psalm": "^4.0"
+    },
+    "repositories": [
+      {
+        "type": "composer",
+        "url": "https://asset-packagist.org"
```

Comment:
do we need this here?

Reply:
Unfortunately yes, as per their documentation:
> Yii relies on [Bower](https://bower.io/) and/or [NPM](https://www.npmjs.com/) packages for the asset (CSS and JavaScript) libraries installation. It uses Composer to obtain these libraries, allowing PHP and CSS/JavaScript package versions to resolve at the same time. This can be achieved either by usage of [asset-packagist.org](https://asset-packagist.org/) or [composer asset plugin](https://github.com/fxpio/composer-asset-plugin).
There may be a workaround, but it doesn't seem immediately obvious to me. Without it, some dependencies of `yiisoft/yii2` fail to resolve.

Reply:
Ok, so we can't do a `composer install` to run tests etc without this. No problem.